app/templates/orders/orders_packages.hbs

```diff
@@ -40,14 +40,23 @@
         </div>
       </div>
     {{else}}
-      {{#link-to 'items.detail' ordersPackage.item.id }}
+      {{#if ordersPackage.item.isSingletonItem}}
```

Comment:
@steveyken , thanks for pointing that out. It definitely reduces the code. `disabled` attribute of link-to just works fine so need to set different route or partial template. I've updated the code. You can check it and let me know if any more changes are required. 
Thanks